components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -545,6 +553,8 @@ - (instancetype)init {
   self = [super init];
   if (self) {
     _internalManager = [[MDCSnackbarManagerInternal alloc] initWithSnackbarManager:self];
+    _uppercaseButtonTitle = YES;
+    _disabledButtonAlpha = 0.12f;
```

Comment:
Nip: we usually do `(CGFloat)0.12` more like a convention on the team. `0.12f` is good to use since no precision is lost.